src/main/java/calculator/exception/PositiveNumbers.java

```diff
@@ -0,0 +1,26 @@
+package calculator.exception;
+
+import calculator.PositiveNumber;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * 0 이상의 숫자들의 리스트를 관리하는 일급 컬렉션
+ */
+public class PositiveNumbers {
+
+    private final List<PositiveNumber> numbers;
+
+    public PositiveNumbers(final List<String> numberToStrings) {
+        this.numbers = numberToStrings.stream()
+            .map(PositiveNumber::new)
+            .collect(Collectors.toList());
```

Comment:
자바 21을 사용하고 계시네요. 

자바 16부터는 이렇게 고칠 수 있어요.

```suggestion
            .toList();
```

Reply:
자바 11 이후의 문법을 아직까지 몰랐는데, 미션을 진행해보면서 틈틈히 공부해봐야겠네요!
꼼꼼한 피드백 감사드려요!

Reply:
2단계 미션에 반영해두겠습니다!